plugins/module_utils/vm.py

```diff
@@ -853,40 +855,87 @@ def get_vm_and_boot_devices(cls, ansible_dict, rest_client):
         ]
         return vm, boot_devices_uuid, boot_devices_ansible
 
-    def update_vm_power_state(self, module, rest_client, desired_power_action):
+    def update_vm_power_state(
+        self, module, rest_client, desired_power_action, ignore_repeated_request: bool
+    ):
         """Sets the power state to what is stored in self.power_state"""
+
         # desired_power_action must be present in FROM_ANSIBLE_TO_HYPERCORE_POWER_ACTION's keys
```

Comment:
This comment is probably for line 876?

Reply:
Also, but primarily it describes method input param. Ty for review.